staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -53,10 +53,31 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	watcherInputBufferSize = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "apiserver_watcher_input_buffer_size",
+			Help: "Watcher buffer size distribution for each resource.",
+			// There are two types of buffers {10, 1000}. Thus we customize buckets
```

Comment:
Do you mean the buffer size falls within a range between 10,1000?

Reply:
There are two buffer types，one 10，one 1000, set the buckets to count them in the same metric.

Reply:
I realize you have removed the metrics, but for future reference I don't think you want to use a histogram here then. If the values for the metric are always 10 or 1000, then all the buckets in the middle would be empty. It would make more sense to use a counter with a label for the buffer size (since we would expect the label values to be constrained to 2 well-known values).